package.json

```diff
@@ -12,6 +12,7 @@
     "test": "vitest run --changed --config ./vitest.config.mjs",
     "test:vite": "yarn test",
     "test:watch": "vitest watch --changed --config ./vitest.config.mjs",
+    "test:all": "vitest run --config ./vitest.config.mjs",
```

Comment:
Thoughts on swapping `test` to be all and `test:changed` for `--changed`?
My personal flow is `yarn test <nameOfFileImWorkingOn>` even if I don't make changes across commits